tests/Monolog/LogProcessorTest.php

```diff
@@ -257,41 +121,6 @@ public function testProcessRecordExceptionBrokenUploader(): void
         self::assertEquals('', $newRecord['channel']);
     }
 
-    public function testProcessRecordDeprecationException(): void
```

Comment:
Why is this test removed?

Reply:
good question :-) .  because the only part in the code where it was being used has been removed.  there is no longer any check for deprecation exceptions (they were previously not being uploaded)

Reply:
I see, so there is now no special handling of deprecation errors. :ok_hand: